samples/nrf9160/azure_fota/src/main.c

```diff
@@ -244,8 +248,25 @@ void main(void)
 	modem_configure();
 	k_sem_take(&network_connected_sem, K_FOREVER);
 
-	static struct azure_iot_hub_config config = {
+#if IS_ENABLED(CONFIG_DEVICE_ID_USE_UUID)
+	char device_id[128];
```

Comment:
UUIDv4 is 36 bytes long, e.g. "67f0ce37-df21-440a-8c8f-ee920430ca02" so this seems a bit large.

Reply:
Yes, will update